src/dependencies.jl

```diff
@@ -356,6 +356,7 @@ function provides{T}(::Type{T},providers::Dict; opts...)
     end
 end
 
+sudoname(c::Cmd) = c == `` ? "" : "sudo "
```

Comment:
how so ? looks like it could be confusing, I wouldn't expect the sudo name to include a space after